content/redirects.txt

```diff
@@ -169,3 +169,6 @@
 # "configuration", because they relate to CLI functionality rather than the
 # configuration language.
 /docs/configuration/environment-variables.html /docs/commands/environment-variables.html
+
+# Load order and semantics is now partially covered by the "overrides" page for 0.12
+/docs/configuration/load.html /docs/configuration/override.html
```

Comment:
Need to hear from someone on core about how to handle this redirect.

Reply:
Nitpick, missing NL?